MIGRATING.md

```diff
@@ -2,6 +2,48 @@
 
 This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
 
+## Migrating `.widgetrc` to `.widgtrc.js`
+
+The rule of thumb is to create a javascript file that `module.exports` the `widgetOptions` in your `.widgetrc` file.
+
+For example, a `.wigetrc` looks like
```

Comment:
Suggestion - utilize the `diff` markdown code syntax to show what needs to change:

```diff
+module.exports = {
- {
 widgetOptions: {
    baseUrl: 'http://localhost:3000',
    logo: '/img/logo_widgico.png',
    logoText: 'Windico',
    features: {
      router: true,
      rememberMe: true,
    },
    // Host the assets (i.e. json files) locally
    assets: {
      baseUrl: '/'
    }
  }
}

```

Reply:
there is no `widgetOptions`